clean/jefferson_so_pprr.py

```diff
@@ -0,0 +1,158 @@
+                     r' ?public assisgments ?| ?youth programs ?| ?community relations ?|'
+                     r' ?lab services ?|\;? ?technical services| ?prisoner ?|'
+                     r' ?asset forfeure ?| ?strategic engagement ?| ?dlr ?|'
+                     r' ?tech services ?| ?sib ?| ?p\.o\.c\. ?| ?school offense ?|'
+                     r' ?accounts payable ?| ?range ?| ?light shop ?| ?public assignments ?|'
+                     r' ?help desk tech ?| ?print shop ?| ?mang services ?|'
+                     r' ?terminal agency ?| ?handgun perms ?| ?insurance ?| ?vice ?|'
+                     r' ?photo lab ?| ?fingerprint ?| ?polygraph | ?evidence ?|'
+                     r' ?digital forensic ?| ?dna ?)', '', regex=True)\
+        .str.replace(r'\bcommaander\b', 'commander', regex=True)\
+        .str.replace(r'\bpropert\b', 'property', regex=True)\
+        .str.replace(r'\bcustodia\b', 'custodian', regex=True)\
+        .str.replace('&', 'and', regex=True)\
+        .str.replace(r'\bcomm\b', 'communications', regex=True)\
+        .str.replace(r'\bdetecti\b', 'detective', regex=True)\
+        .str.replace('booking of', 'booking officer', regex=False)\
+        .str.replace(r'^(patrol)? ?deputy patrol ?(deputy)?$', 'patrol deputy', regex=True)\
+        .str.replace(r' \- ', '; ', regex=True)\
+        .str.replace(r'\bsupe?rv?i?s?o?\:?\b', 'supervisor', regex=True)\
+        .str.replace(r'^r\.f\.manag', 'r.f manager', regex=True)\
+        .str.replace(r'(^of$|^cc$)', '', regex=True)\
+        .str.replace(r'^against$', 'against persons', regex=True)\
+        .str.replace('gretnadetective', 'gretna detective', regex=False)\
+        .str.replace('fbidetective', 'fbi detective', regex=False)\
+        .str.replace(r'\bcomm?a??n?d?e?r?i?o?\b', 'commander', regex=True)\
+        .str.replace('chief officer', 'chief officer', regex=False)\
+        .str.replace(r'^invest$', 'investigator', regex=True)\
+        .str.replace(r'^\.? ?', '', regex=True)\
+        .str.replace(r'asst\.? ?', 'assistant ', regex=True)\
+        .str.replace('systspeciali', 'systems specialist', regex=False)\
+        .str.replace(r'^and ?', '', regex=True)\
+        .str.replace('digal', 'digital', regex=False)\
+        .str.replace('emergencycomputer', 'emergency computer', regex=False)\
+        .str.replace(r'^serge$', 'sergeant', regex=True)\
+        .str.replace(r'^detective detective$', 'detective', regex=True)\
+        .str.replace(r'^sergeant patrol sergeant$', 'patrol sergeant', regex=True)\
+        .str.replace(r'\bpatroi\b', 'patrol', regex=True)\
+        .str.replace(r'\:$', '', regex=True)\
+        .str.replace('commanderssary', 'commander', regex=False)\
+        .str.replace(r' ?\-$', '', regex=True)\
+        .str.replace('coordinato', 'coordinator', regex=False)\
+        .str.replace(r'^repairman repairman$', 'repairman', regex=True)\
+        .str.replace('systmanager', 'systems manager', regex=False)
```

Comment:
There's a very good use case for lookup table right here.

Reply:
Sounds good. I'll make the change.